posts/privilege_bingo.md

```diff
@@ -0,0 +1,53 @@
+# Privilege bingo
+
+Intersectionality, BAME, cis, anonymous hiring, inclusion. These words are likely not on your ticket if you are playing privilege bingo. Like a canary in a coal mine, a lack of understanding of these concepts will likely mean your daily actions reinforce systemic discrimination.
+
+We aren't good at diversity and that is something we need to work on as a business, as an industry, and as a collection of humans. There are direct and indirect effects caused by our policies and our privilege that actively work against us building a more diverse team.
+
+We didn't set out to avoid diversity, we just didn't and perhaps still don't have enough knowledge to encourage it. We didn't implement a refer a friend scheme to discourage diverse hiring, however it's fairly well known that your friends, especially ex-colleagues will mostly be like you in terms of gender and ethnicity given you're in an industry already suffering from a lack of diversity.
+
+## Awakening from ignorance
+
+Ignorance is bliss right? I'm unsure that's right but it's certainly easier. Through slow osmosis, from forgiving friends offering words of advice, and by growing noise about diversity, we are slowly waking up as an organisation.
+
+> The first step toward change is awareness. The second step is acceptance.
+>
+> ~ Nathaniel Brandon
+
+The first step has to be awareness of the fact we're poor at diversity. I mean it's hard to argue looking at the fact we're almost all white guys. The next step is acceptance, a willingness to come to terms with reality without protest or reaction. Lastly, action.
+
+## Identifying smells
+
+Coming back to the refer a friend scheme, why did we implement that scheme? We actually did it to encourage more of the same types of people joining. It's not that we wanted to encourage more of the same ethnicity, gender, or other intersections. We did it because we needed to hire quickly, and hiring people recommend by those we already know and trust is easier than the alternatives.
+
+With this ease, comes a problem. The issue is that there are direct consequences of enacting a policy like this. You will immediately be discouraging diversity by inviting more of the same gender and ethnicity unless you have previously worked in diverse organisations. You are also rewarding and reinforcing this kind of behaviour in your organisation.
+
+Smells are often a little difficult to work out, especially being so naive to privilege. Detecting the smells of exclusionary practices, much like code smells, gets easier with experience I am sure.
+
+## Accepting failures
+
+The tougher part is to gain wider support. Coming back to the privilege bingo, when a team isn't educated in areas of diversity, it is difficult to make diversity a high priority. Sure, diversity is gaining wider attention and people are talking the talk, but walking the walk is a whole other thing.
```

Comment:
Hmmm, no, I guess you are right. Perhaps education of HOW to tackle the issue is missing. I think I read it as education of diversity being a problem in the first place, which I might also be wrong about. I think I was maybe reading this from a woman's perspective, instead of a business trying to make diversity a priority, but as that is the whole focus point of the blog, you were right in the first place.